tests/core/pyspec/eth2spec/test/altair/transition/test_activation.py

```diff
@@ -0,0 +1,228 @@
+import random
```

Comment:
change the name of this file to something like `test_activations_and_exits.py`?

it wasn't clear until i dug into the code that this file also was testing the exit logic